services/sts/src/test/java/software/amazon/awssdk/services/sts/internal/StsWebIdentityCredentialsProviderFactoryTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.sts.internal;
+
+import org.junit.Assert;
+import org.junit.Test;
+import software.amazon.awssdk.auth.credentials.WebIdentityTokenCredentialsProviderFactory;
+import software.amazon.awssdk.auth.credentials.internal.WebIdentityCredentialsUtils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+public class StsWebIdentityCredentialsProviderFactoryTest {
+
+
+    @Test
+    public void stsWebIdentityCredentialsProviderFactory_with_webIdentityCredentialsUtils(){
+        final WebIdentityTokenCredentialsProviderFactory factory = WebIdentityCredentialsUtils.factory();
```

Comment:
nit: we don't use `final` on local variables. We have a checkstyle for it, but it doesn't apply to tests

Reply:
done.